### ISSUE TITLE
Server, compiler and project configs. 

### ISSUE THREAD

**Issue by fakuivan** [JulienKluge/Spedit#54]
It would be cool to have separed |server upload-rcon|, |compiler and sourcemod library directory| configs. So that way you could switch to different compiler versions and servers without having to re-type all of those specific configurations. That way you just have to select a server and compiler config from the project configuration window

**Comment by JulienKluge** [comment]
well, you could just create a different config and save it.i mean, all you specified is supported by the current config system if I understood you right.

**Comment by fakuivan** [comment]
yep, it is. The problem comes up when you frequently jump from test server to test server and need to change configs regularly, or if you want to use a different compiler.
